rs/nns/integration_tests/BUILD.bazel

```diff
@@ -391,7 +391,8 @@ rust_ic_test(
     env = DEV_ENV,
     proc_macro_deps = MACRO_DEPENDENCIES + MACRO_DEV_DEPENDENCIES,
     tags = [
-        "manual",  # CI should not be downloading nns_state.tar.zst.
+        "nns_tests_nightly",  # Run this test in the nns-tests-nightly GitHub Action job.
```

Comment:
I would not mind this being run more frequently (e.g. hourly). FWIW, it only takes 5 minutes (on my devenv vm). Ofc, it does use a fair amount of disk (10 GB or so).

Reply:
Let's first try this out daily and if that proves stable we move it to hourly.